<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_port_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>EPC 号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_epc_number">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_epc_get">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>获取EPC号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_epc_write">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>写入EPC号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_epc_cancel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>取消写入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_epc_read">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>读取EPC号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>输入框:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_ic">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>IC卡号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ic_number">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_card">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>150</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>确认开卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh_serial">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>打印机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_conn_print">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>端 口:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="com_ports">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_conn_serial">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>卡包号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_card_bag_number">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>400</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="enable_epc">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>开EPC卡</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="disable_epc">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>不开EPC卡</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
